data_collection/gazette/spiders/rj_sao_goncalo.py

```diff
@@ -0,0 +1,9 @@
+from gazette.spiders.base.instar import BaseInstarSpider
+
+
+class RjSaoGoncaloSpider(BaseInstarSpider):
+
+    TERRITORY_ID = "3304904"
+    name = "rj_sao_goncalo"
+    allowed_domains = ["www.saogoncalo.rj.gov.br"]
+    start_urls = ["https://servicos.pmsg.rj.gov.br/diario_oficial"]
```

Comment:
This page doesn't exist but this one does https://servicos.pmsg.rj.gov.br/diario_oficial.php. But after running it with this one and with https://www.saogoncalo.rj.gov.br/diario-oficial/, none are working.
By looking at other portals like http://www.altoalegre.sp.gov.br/portal/diario-oficial that also implement `BaseInstarSpider`, it seems that São Gonçalo isn't using Instar. Could you check this, please? @joaopclima